zeppelin-server/src/main/resources/zeppelin-site.xml

```diff
@@ -0,0 +1,508 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+<configuration>
+
+<property>
+  <name>zeppelin.server.addr</name>
+  <value>0.0.0.0</value>
+  <description>Server address</description>
+</property>
+
+<property>
+  <name>zeppelin.server.port</name>
+  <value>8080</value>
+  <description>Server port.</description>
+</property>
+
+<property>
+  <name>zeppelin.server.ssl.port</name>
+  <value>8443</value>
+  <description>Server ssl port. (used when ssl property is set to true)</description>
+</property>
+
+<property>
+  <name>zeppelin.server.context.path</name>
+  <value>/</value>
+  <description>Context Path of the Web Application</description>
+</property>
+
+<property>
+  <name>zeppelin.war.tempdir</name>
+  <value>webapps</value>
+  <description>Location of jetty temporary directory</description>
+</property>
+
+<property>
+  <name>zeppelin.notebook.dir</name>
+  <value>notebook</value>
+  <description>path or URI for notebook persist</description>
+</property>
+
+<property>
+  <name>zeppelin.notebook.homescreen</name>
+  <value></value>
+  <description>id of notebook to be displayed in homescreen. ex) 2A94M5J1Z Empty value displays default home screen</description>
+</property>
+
+<property>
+  <name>zeppelin.notebook.homescreen.hide</name>
+  <value>false</value>
+  <description>hide homescreen notebook from list when this value set to true</description>
+</property>
+
+
+<!-- Amazon S3 notebook storage -->
+<!-- Creates the following directory structure: s3://{bucket}/{username}/{notebook-id}/note.json -->
```

Comment:
we haven't need this file before, why do we need it now?
also why are lines commented out?

Reply:
The file is copied to this path because I had troubles running the `ZeppelinServer` locally. It was missing this file.

Those lines are commented as it is copied from `conf/zeppelin-site.xml.template` https://github.com/apache/zeppelin/blob/master/conf/zeppelin-site.xml.template#L71-L73.

Reply:
I'm wondering how other would think about this - I think we don't include default xml config (and only .xml.template instead) for a reason

Reply:
I understand your point. I would suggest removing this file and adda step to the README to copy this file and do the proper configuration changes locally when running `ZeppelinServer` in development mode.

Reply:
ok

Reply:
👍 

1. Removed the `zeppelin-site.xml` from Zeppelin Server resources
2. Added instructions to the `how_to_contribute.md` to copy the template file from `conf` directory to resources directory
3. Added the `zeppelin-site.xml` in the resources directory to `.gitignore`

https://github.com/apache/zeppelin/pull/2700/commits/e98d1b0f2c1bb51c261807365f5c3d0d31a7d46c and https://github.com/apache/zeppelin/pull/2700/commits/b4fe44d7453dbb734910f06a04d45a5657364ae7.